packages/components/src/Select/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Select as SelectLegacy } from "./Select";
+import { SelectRebuilt } from "./Select.rebuilt";
+import { SelectProps, SelectRebuiltProps } from "./Select.types";
+
+export { Option } from "./Option";
+export type SelectShimProps = SelectProps | SelectRebuiltProps;
```

Comment:
I think we should expose `SelectProps` as `SelectLegacyProps` and `SelectRebuiltProps` to help with people who want to write components that use the `SelectProps` in their own components. We did the same with `InputText` because there were some issues with typescript when people used `React.ComponentProps<ComponentThatWasShimmed>`

Reply:
We did this here
https://github.com/GetJobber/atlantis/blob/JOB-120388-do-it-select/packages/components/src/InputText/index.tsx#L36